src/Juvix/Syntax/Concrete/Scoped/Error/Types.hs

```diff
@@ -522,3 +522,27 @@ ambiguousMessage n es =
       <> "It could be any of:"
       <> line
       <> indent' (vsep (map ppCode es))
+
+newtype WrongInductiveParameterName = WrongInductiveParameterName
+  { _wrongInductiveParameterName :: Symbol
+  }
+  deriving stock (Show)
+
+makeLenses ''WrongInductiveParameterName
+
+instance ToGenericError WrongInductiveParameterName where
+  genericError e =
+    GenericError
+      { _genericErrorLoc = i,
+        _genericErrorMessage = prettyError msg,
+        _genericErrorIntervals = [i]
+      }
+    where
+      param = e ^. wrongInductiveParameterName
+      i = getLoc param
+      msg =
+        "The second occurrence of the parameter name"
```

Comment:
I would say "Invalid name X. Inductive parameter names cannot be repeated". I feel it is more concise and easier to understand